CONTRIBUTING.md

```diff
@@ -0,0 +1,235 @@
+Contributing to InfluxDB v2
+========================
+
+Bug reports
+---------------
+Before you file an issue, please search existing issues in case it has already been filed, or perhaps even fixed. If you file an issue, please include the following.
+* Full details of your operating system (or distribution) e.g. 64-bit Ubuntu 14.04.
+* The version of InfluxDB you are running
+* Whether you installed it using a pre-built package, or built it from source.
+* If applicable, a small test case or method to reproduce the issue described.
+
+Remember the golden rule of bug reports: **The easier you make it for us to reproduce the problem, the faster it will get fixed.**
+If you have never written a bug report before, or if you want to brush up on your bug reporting skills, we recommend reading [Simon Tatham's essay "How to Report Bugs Effectively."](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html)
+
+TEST CASES SECTION NEEDS REVISION based on v2 API
+
+Test cases should be in the form of `curl` commands. For example:
+```bash
+# write data
+curl -XPOST "http://localhost:9999/api/v2/write?org=YOUR_ORG&bucket=YOUR_BUCKET&precision=s" \
+  --header "Authorization: Token YOURAUTHTOKEN" \
+  --data-raw "mem,host=host1 used_percent=23.43234543 1556896326"
+
+# query data
+# Bug: expected it to return no data, but data comes back.
+curl http://localhost:9999/api/v2/query?org=my-org -XPOST -sS \
+  -H 'Authorization: Token YOURAUTHTOKEN' \
+  -H 'Accept: application/csv' \
+  -H 'Content-type: application/vnd.flux' \
+  -d 'from(bucket:"example-bucket")
+        |> range(start:-1000h)
+        |> group(columns:["_measurement"], mode:"by")
+        |> sum()'
+```
+**If you don't include a clear test case like this, your issue may not be investigated, and may even be closed**. If writing the data is too difficult, please zip up your data directory and include a link to it in your bug report.
+
+Please note that issues are *not the place to file general questions* such as "how do I use collectd with InfluxDB?" Questions of this nature should be sent to the [InfluxData Community](https://community.influxdata.com/), not filed as issues. Issues like this will be closed.
+
+Feature requests
+---------------
+We really like to receive feature requests as it helps us prioritize our work. Please be clear about your requirements, as incomplete feature requests may simply be closed if we don't understand what you would like to see added to InfluxDB.  If you find your feature request already exists as a Github issue please indicate your support for that feature by using the "thumbs up" reaction.
```

Comment:
Why does every section seem to describe our right to close issues?

So far I've read:
1. send perfect curl requests 
2. send perfect bug reports
3. send perfect feature requests

or else!

Sure, I understand you want to describe that we may close issues.  If it is that important perhaps its own section.

Reply:
Fixed in the latest version.